appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
@mark-friedman Hey have you gotten a chance to look this over? I'll be working on it for a good amount of time later today and would really appreciate some feedback/help. Thanks!